R/JaBbA.R

```diff
@@ -4004,50 +4151,50 @@ read.junctions = function(rafile, keep.features = T, seqlengths = hg_seqlengths(
                           get.loose = FALSE ## if TRUE will return a list with fields $junctions and $loose.ends
                           )
 {
-  if (is.character(rafile))
```

Comment:
There's some problem in `read.junctions` causing the examples to fail via `devtools::check()`. This should be fixed, and then the Travis build should pass. 

Also, the spacing looks off for `read.junctions()`